lib/tasks/test.rb

```diff
@@ -18,6 +18,7 @@
   task :setup do
     testing.each_pair do |app, _task|
       chdir PROJ_DIR.join(app.to_s) do
+        sh "yarn install" if app.to_s == "apps/dashboard"
```

Comment:
Look in main repo Rakefile.  If we do indeed have to stay with using yarn then add a function like this:

```
def yarn_app?
  @path.join('yarn.lock').exist?
end
```
Also this install step should not be in this rake library file. It should be in `Rakefile` at root of repo as this is a build step and that's where all build steps are stored.

Reply:
This is now executed in bin/setup and a yarn local to the app is installed.

Reply:
I was wrong. Rather why can't we do `PASSENGER_ENV=development bin/setup` in `task :setup` instead?